src/app/pages/PerpetualPage/components/TradeForm/index.tsx

```diff
@@ -250,6 +233,36 @@ export const TradeForm: React.FC<ITradeFormProps> = ({
     [onChange, trade],
   );
 
+  useEffect(() => {
+    if (!limit || bignumber(limit).lessThan(1)) {
```

Comment:
This could cause issues on future pairs
```suggestion
    if (!limit || bignumber(limit).lessThanOrEqualTo(0)) {
```